### ISSUE TITLE
Error when getting /api/apps/overview?name=app-name

### ISSUE THREAD

**Issue by allburov** [texm/shokku#9]
Hi! For a single app we're getting the error when open "Overview app" page
There's a error in developer console (chrome) for the request `GET /api/apps/overview?name=app-name`
```json
{"fields":{"Name":"max"},"type":"validation"}
```

**Comment by allburov** [comment]
With other 3 apps it works with no issue. 

Let me know if you need more details about that or if you could guide me to the right place in the repo where I could create the fix for it with a PR!

**Comment by allburov** [comment]
No errors in the docker container, only this
```
2024-01-18T12:44:18.635481231Z app[web.1]: {"level":"info","remote_ip":"51.195.41.143","latency":"104.351µs","host":"shokku.domain.tech","request":"GET /api/apps/overview?name=app","status":400,"size":46,"time":1705581858,"message":"Client error"}
```

**Comment by texm** [comment]
The fix from PR #10 has been released as `ghcr.io/texm/shokku:v0.0.3`

Instructions for upgrading are here, if needed : https://shokku.dev/docs/upgrading

**Comment by allburov** [comment]
@texm thank you! it works :rocket:
